--- /proc/self/fd/16	2023-11-25 18:53:22.356724166 +0900
+++ /proc/self/fd/21	2023-11-25 18:53:22.356724166 +0900
@@ -16,28 +16,7 @@
     "description": "この春の新作コスメやおすすめのアイテムを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/8/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 29,
-        "name": "ピアノ"
-      },
-      {
-        "id": 46,
-        "name": "映画レビュー"
-      },
-      {
-        "id": 86,
-        "name": "仮想通貨"
-      },
-      {
-        "id": 1,
-        "name": "ライブ配信"
-      },
-      {
-        "id": 80,
-        "name": "ライフハック"
-      }
-    ],
+    "tags": null,
     "start_at": 1708441200,
     "end_at": 1708448400
   },
@@ -58,28 +37,7 @@
     "description": "庭やベランダでのガーデニングの様子を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/11/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 58,
-        "name": "ペットと一緒"
-      },
-      {
-        "id": 15,
-        "name": "ゲーム解説"
-      },
-      {
-        "id": 45,
-        "name": "アニメトーク"
-      },
-      {
-        "id": 103,
-        "name": "椅子"
-      },
-      {
-        "id": 31,
-        "name": "DJセット"
-      }
-    ],
+    "tags": null,
     "start_at": 1707804000,
     "end_at": 1707858000
   },
@@ -100,28 +58,7 @@
     "description": "手軽にできる料理を実際に作りながら紹介していきます！",
     "playlist_url": "https://media.xiii.isucon.dev/api/8/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 63,
-        "name": "テクノロジー"
-      },
-      {
-        "id": 53,
-        "name": "ヨガ"
-      },
-      {
-        "id": 39,
-        "name": "手料理"
-      },
-      {
-        "id": 103,
-        "name": "椅子"
-      },
-      {
-        "id": 49,
-        "name": "メイク"
-      }
-    ],
+    "tags": null,
     "start_at": 1708437600,
     "end_at": 1708452000
   },
@@ -142,28 +79,7 @@
     "description": "様々な資格試験に合格するための勉強方法を伝授します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 98,
-        "name": "コラボ配信"
-      },
-      {
-        "id": 3,
-        "name": "生放送"
-      },
-      {
-        "id": 4,
-        "name": "アドバイス"
-      },
-      {
-        "id": 100,
-        "name": "生誕祭"
-      },
-      {
-        "id": 81,
-        "name": "教育"
-      }
-    ],
+    "tags": null,
     "start_at": 1708365600,
     "end_at": 1708372800
   },
@@ -184,28 +100,7 @@
     "description": "日常で使える基本的な英会話フレーズを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/8/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 36,
-        "name": "趣味の話"
-      },
-      {
-        "id": 102,
-        "name": "サプライズ"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 88,
-        "name": "不動産"
-      },
-      {
-        "id": 87,
-        "name": "株式投資"
-      }
-    ],
+    "tags": null,
     "start_at": 1707786000,
     "end_at": 1707847200
   },
@@ -226,28 +121,7 @@
     "description": "マジシャンが実際の手品を披露し、その秘密を少しだけ公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/1/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 10,
-        "name": "FPS"
-      },
-      {
-        "id": 18,
-        "name": "新情報発表"
-      },
-      {
-        "id": 67,
-        "name": "ニュース解説"
-      },
-      {
-        "id": 83,
-        "name": "ビジネス"
-      },
-      {
-        "id": 98,
-        "name": "コラボ配信"
-      }
-    ],
+    "tags": null,
     "start_at": 1707116400,
     "end_at": 1707159600
   },
@@ -268,28 +142,7 @@
     "description": "朝の時間を有効に使いたい皆さん、一緒に活動を始めましょう！",
     "playlist_url": "https://media.xiii.isucon.dev/api/3/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 77,
-        "name": "サッカー"
-      },
-      {
-        "id": 57,
-        "name": "キャンプ"
-      },
-      {
-        "id": 65,
-        "name": "プログラミング"
-      },
-      {
-        "id": 45,
-        "name": "アニメトーク"
-      },
-      {
-        "id": 51,
-        "name": "健康"
-      }
-    ],
+    "tags": null,
     "start_at": 1707112800,
     "end_at": 1707148800
   },
@@ -310,28 +163,7 @@
     "description": "実際に起業を経験したゲストと、成功の秘訣や失敗談について語り合います。",
     "playlist_url": "https://media.xiii.isucon.dev/api/5/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 60,
-        "name": "犬"
-      },
-      {
-        "id": 71,
-        "name": "心理学"
-      },
-      {
-        "id": 41,
-        "name": "アート配信"
-      },
-      {
-        "id": 69,
-        "name": "文化"
-      },
-      {
-        "id": 25,
-        "name": "アコースティック"
-      }
-    ],
+    "tags": null,
     "start_at": 1707102000,
     "end_at": 1707138000
   },
@@ -352,28 +184,7 @@
     "description": "子育ての専門家が、子供の健康や教育に関する悩みや質問に答えます。",
     "playlist_url": "https://media.xiii.isucon.dev/api/11/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 54,
-        "name": "ダンス"
-      },
-      {
-        "id": 45,
-        "name": "アニメトーク"
-      },
-      {
-        "id": 78,
-        "name": "野球"
-      },
-      {
-        "id": 75,
-        "name": "コメディ"
-      },
-      {
-        "id": 93,
-        "name": "オカルト"
-      }
-    ],
+    "tags": null,
     "start_at": 1707076800,
     "end_at": 1707138000
   },
@@ -394,28 +205,7 @@
     "description": "新しいRPGゲームを実況しながら進めていきます！みんなでワイワイ攻略しよう！",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 23,
-        "name": "カバーソング"
-      },
-      {
-        "id": 94,
-        "name": "UFO"
-      },
-      {
-        "id": 12,
-        "name": "対戦ゲーム"
-      }
-    ],
+    "tags": null,
     "start_at": 1707069600,
     "end_at": 1707112800
   },
@@ -436,28 +226,7 @@
     "description": "一緒に美味しい手作りクッキーを作る時間。簡単なレシピを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/4/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 73,
-        "name": "科学"
-      },
-      {
-        "id": 34,
-        "name": "夜ふかし"
-      },
-      {
-        "id": 103,
-        "name": "椅子"
-      },
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 31,
-        "name": "DJセット"
-      }
-    ],
+    "tags": null,
     "start_at": 1707058800,
     "end_at": 1707116400
   },
@@ -478,28 +247,7 @@
     "description": "新作のアクションゲームの実況プレイを行います！",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 61,
-        "name": "釣り"
-      },
-      {
-        "id": 93,
-        "name": "オカルト"
-      },
-      {
-        "id": 15,
-        "name": "ゲーム解説"
-      },
-      {
-        "id": 6,
-        "name": "プロゲーマー"
-      },
-      {
-        "id": 90,
-        "name": "スピリチュアル"
-      }
-    ],
+    "tags": null,
     "start_at": 1708358400,
     "end_at": 1708365600
   },
@@ -520,28 +268,7 @@
     "description": "実際の絵画の制作過程をリアルタイムで公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/3/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 103,
-        "name": "椅子"
-      },
-      {
-        "id": 26,
-        "name": "歌配信"
-      },
-      {
-        "id": 20,
-        "name": "チャット交流"
-      },
-      {
-        "id": 11,
-        "name": "アクションゲーム"
-      },
-      {
-        "id": 34,
-        "name": "夜ふかし"
-      }
-    ],
+    "tags": null,
     "start_at": 1707040800,
     "end_at": 1707091200
   },
@@ -562,28 +289,7 @@
     "description": "日常の出来事やニュースをユーモアたっぷりに解説します！笑顔で過ごしましょう！",
     "playlist_url": "https://media.xiii.isucon.dev/api/3/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 65,
-        "name": "プログラミング"
-      },
-      {
-        "id": 71,
-        "name": "心理学"
-      },
-      {
-        "id": 39,
-        "name": "手料理"
-      }
-    ],
+    "tags": null,
     "start_at": 1707033600,
     "end_at": 1707102000
   },
@@ -604,28 +310,7 @@
     "description": "ダンスの特訓の様子をリアルタイムで公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/7/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 2,
-        "name": "ゲーム実況"
-      },
-      {
-        "id": 40,
-        "name": "レシピ紹介"
-      },
-      {
-        "id": 12,
-        "name": "対戦ゲーム"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 88,
-        "name": "不動産"
-      }
-    ],
+    "tags": null,
     "start_at": 1708336800,
     "end_at": 1708358400
   },
@@ -646,28 +331,7 @@
     "description": "歴史学者が、戦国時代の英雄たちやその背景を詳しく解説します",
     "playlist_url": "https://media.xiii.isucon.dev/api/1/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 90,
-        "name": "スピリチュアル"
-      },
-      {
-        "id": 19,
-        "name": "Q&Aセッション"
-      },
-      {
-        "id": 16,
-        "name": "ホラーゲーム"
-      },
-      {
-        "id": 97,
-        "name": "ファンミーティング"
-      },
-      {
-        "id": 18,
-        "name": "新情報発表"
-      }
-    ],
+    "tags": null,
     "start_at": 1707022800,
     "end_at": 1707069600
   },
@@ -688,28 +352,7 @@
     "description": "プロの画家が、風景画の基本的な描き方や技術を伝授します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/4/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 98,
-        "name": "コラボ配信"
-      },
-      {
-        "id": 37,
-        "name": "語学学習"
-      },
-      {
-        "id": 52,
-        "name": "ワークアウト"
-      },
-      {
-        "id": 13,
-        "name": "マルチプレイ"
-      },
-      {
-        "id": 57,
-        "name": "キャンプ"
-      }
-    ],
+    "tags": null,
     "start_at": 1707004800,
     "end_at": 1707076800
   },
@@ -730,28 +373,7 @@
     "description": "プロのカメラマンが、写真の基本的な撮影技術やコツを伝授します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 14,
-        "name": "シングルプレイ"
-      },
-      {
-        "id": 39,
-        "name": "手料理"
-      },
-      {
-        "id": 102,
-        "name": "サプライズ"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 89,
-        "name": "キャリア"
-      }
-    ],
+    "tags": null,
     "start_at": 1706968800,
     "end_at": 1707030000
   },
@@ -772,28 +394,7 @@
     "description": "今月読んだ本の中から特に感動した一冊を紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 50,
-        "name": "ビューティー"
-      },
-      {
-        "id": 33,
-        "name": "朝活"
-      },
-      {
-        "id": 6,
-        "name": "プロゲーマー"
-      },
-      {
-        "id": 58,
-        "name": "ペットと一緒"
-      },
-      {
-        "id": 74,
-        "name": "マジック"
-      }
-    ],
+    "tags": null,
     "start_at": 1708329600,
     "end_at": 1708362000
   },
@@ -814,28 +415,7 @@
     "description": "最新のアニメについて、感想や考察をみんなで交換しましょう！",
     "playlist_url": "https://media.xiii.isucon.dev/api/1/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 22,
-        "name": "音楽ライブ"
-      },
-      {
-        "id": 96,
-        "name": "コンサート"
-      },
-      {
-        "id": 70,
-        "name": "社会問題"
-      },
-      {
-        "id": 33,
-        "name": "朝活"
-      },
-      {
-        "id": 35,
-        "name": "日常話"
-      }
-    ],
+    "tags": null,
     "start_at": 1706961600,
     "end_at": 1707033600
   },
@@ -856,28 +436,7 @@
     "description": "都市の喧騒を離れ、美しい星空を観測します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/5/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 7,
-        "name": "新作ゲーム"
-      },
-      {
-        "id": 59,
-        "name": "猫"
-      },
-      {
-        "id": 24,
-        "name": "オリジナル楽曲"
-      },
-      {
-        "id": 42,
-        "name": "絵描き"
-      },
-      {
-        "id": 6,
-        "name": "プロゲーマー"
-      }
-    ],
+    "tags": null,
     "start_at": 1708308000,
     "end_at": 1708336800
   },
@@ -898,28 +457,7 @@
     "description": "新しいRPGゲームを実況しながら進めていきます！みんなでワイワイ攻略しよう！",
     "playlist_url": "https://media.xiii.isucon.dev/api/7/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 45,
-        "name": "アニメトーク"
-      },
-      {
-        "id": 35,
-        "name": "日常話"
-      },
-      {
-        "id": 81,
-        "name": "教育"
-      },
-      {
-        "id": 22,
-        "name": "音楽ライブ"
-      },
-      {
-        "id": 37,
-        "name": "語学学習"
-      }
-    ],
+    "tags": null,
     "start_at": 1708297200,
     "end_at": 1708322400
   },
@@ -940,28 +478,7 @@
     "description": "子育ての専門家が、子供の健康や教育に関する悩みや質問に答えます。",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 65,
-        "name": "プログラミング"
-      },
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 95,
-        "name": "都市伝説"
-      },
-      {
-        "id": 83,
-        "name": "ビジネス"
-      },
-      {
-        "id": 77,
-        "name": "サッカー"
-      }
-    ],
+    "tags": null,
     "start_at": 1706947200,
     "end_at": 1706990400
   },
@@ -982,28 +499,7 @@
     "description": "美容師が春の新作ヘアスタイルやカラーを実際に公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 41,
-        "name": "アート配信"
-      },
-      {
-        "id": 36,
-        "name": "趣味の話"
-      },
-      {
-        "id": 90,
-        "name": "スピリチュアル"
-      },
-      {
-        "id": 99,
-        "name": "記念配信"
-      },
-      {
-        "id": 85,
-        "name": "投資"
-      }
-    ],
+    "tags": null,
     "start_at": 1706940000,
     "end_at": 1706997600
   },
@@ -1024,28 +520,7 @@
     "description": "夜の時間を、心地良い音楽とともに過ごしましょう。",
     "playlist_url": "https://media.xiii.isucon.dev/api/2/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 85,
-        "name": "投資"
-      },
-      {
-        "id": 14,
-        "name": "シングルプレイ"
-      },
-      {
-        "id": 91,
-        "name": "占い"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 16,
-        "name": "ホラーゲーム"
-      }
-    ],
+    "tags": null,
     "start_at": 1708275600,
     "end_at": 1708308000
   },
@@ -1066,28 +541,7 @@
     "description": "筋トレの基本的な方法やコツを紹介しながら、実際にトレーニングを行います。",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 79,
-        "name": "バスケットボール"
-      },
-      {
-        "id": 80,
-        "name": "ライフハック"
-      },
-      {
-        "id": 51,
-        "name": "健康"
-      },
-      {
-        "id": 100,
-        "name": "生誕祭"
-      },
-      {
-        "id": 9,
-        "name": "RPG"
-      }
-    ],
+    "tags": null,
     "start_at": 1706932800,
     "end_at": 1706968800
   },
@@ -1108,28 +562,7 @@
     "description": "キャンプ場からのライブ！自然の中での生活や料理を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 74,
-        "name": "マジック"
-      },
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 91,
-        "name": "占い"
-      },
-      {
-        "id": 7,
-        "name": "新作ゲーム"
-      },
-      {
-        "id": 27,
-        "name": "楽器演奏"
-      }
-    ],
+    "tags": null,
     "start_at": 1706932800,
     "end_at": 1707008400
   },
@@ -1150,28 +583,7 @@
     "description": "手相をもとに、あなたの未来や運命を占います。",
     "playlist_url": "https://media.xiii.isucon.dev/api/3/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 13,
-        "name": "マルチプレイ"
-      },
-      {
-        "id": 20,
-        "name": "チャット交流"
-      },
-      {
-        "id": 87,
-        "name": "株式投資"
-      },
-      {
-        "id": 12,
-        "name": "対戦ゲーム"
-      },
-      {
-        "id": 77,
-        "name": "サッカー"
-      }
-    ],
+    "tags": null,
     "start_at": 1706907600,
     "end_at": 1706961600
   },
@@ -1192,28 +604,7 @@
     "description": "新作のアクションゲームの実況プレイを行います！",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 67,
-        "name": "ニュース解説"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 98,
-        "name": "コラボ配信"
-      },
-      {
-        "id": 89,
-        "name": "キャリア"
-      },
-      {
-        "id": 90,
-        "name": "スピリチュアル"
-      }
-    ],
+    "tags": null,
     "start_at": 1706893200,
     "end_at": 1706929200
   },
@@ -1234,28 +625,7 @@
     "description": "歴史学者が、戦国時代の英雄たちやその背景を詳しく解説します",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 5,
-        "name": "初心者歓迎"
-      },
-      {
-        "id": 25,
-        "name": "アコースティック"
-      },
-      {
-        "id": 77,
-        "name": "サッカー"
-      },
-      {
-        "id": 32,
-        "name": "トーク配信"
-      },
-      {
-        "id": 43,
-        "name": "DIY"
-      }
-    ],
+    "tags": null,
     "start_at": 1708275600,
     "end_at": 1708297200
   },
@@ -1276,28 +646,7 @@
     "description": "車の基本的なメンテナンス方法や作業のポイントを専門家が解説します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/3/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 60,
-        "name": "犬"
-      },
-      {
-        "id": 29,
-        "name": "ピアノ"
-      },
-      {
-        "id": 75,
-        "name": "コメディ"
-      },
-      {
-        "id": 21,
-        "name": "視聴者参加"
-      },
-      {
-        "id": 9,
-        "name": "RPG"
-      }
-    ],
+    "tags": null,
     "start_at": 1706875200,
     "end_at": 1706940000
   },
@@ -1318,28 +667,7 @@
     "description": "ゴルフのトーナメントの進行や選手の動向をリアルタイムで実況します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 42,
-        "name": "絵描き"
-      },
-      {
-        "id": 90,
-        "name": "スピリチュアル"
-      },
-      {
-        "id": 41,
-        "name": "アート配信"
-      },
-      {
-        "id": 10,
-        "name": "FPS"
-      },
-      {
-        "id": 84,
-        "name": "起業"
-      }
-    ],
+    "tags": null,
     "start_at": 1706853600,
     "end_at": 1706929200
   },
@@ -1360,28 +688,7 @@
     "description": "都市の喧騒を離れ、美しい星空を観測します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/7/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 7,
-        "name": "新作ゲーム"
-      },
-      {
-        "id": 61,
-        "name": "釣り"
-      },
-      {
-        "id": 27,
-        "name": "楽器演奏"
-      },
-      {
-        "id": 13,
-        "name": "マルチプレイ"
-      },
-      {
-        "id": 41,
-        "name": "アート配信"
-      }
-    ],
+    "tags": null,
     "start_at": 1706846400,
     "end_at": 1706889600
   },
@@ -1402,28 +709,7 @@
     "description": "今日あった日常の出来事やハイライトを話していきます。",
     "playlist_url": "https://media.xiii.isucon.dev/api/4/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 55,
-        "name": "旅行記"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 31,
-        "name": "DJセット"
-      },
-      {
-        "id": 93,
-        "name": "オカルト"
-      },
-      {
-        "id": 10,
-        "name": "FPS"
-      }
-    ],
+    "tags": null,
     "start_at": 1708272000,
     "end_at": 1708286400
   },
@@ -1444,28 +730,7 @@
     "description": "手芸好きのためのライブ。今回はかわいい刺繍の作り方を紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 58,
-        "name": "ペットと一緒"
-      },
-      {
-        "id": 67,
-        "name": "ニュース解説"
-      },
-      {
-        "id": 46,
-        "name": "映画レビュー"
-      },
-      {
-        "id": 60,
-        "name": "犬"
-      },
-      {
-        "id": 30,
-        "name": "バンドセッション"
-      }
-    ],
+    "tags": null,
     "start_at": 1706835600,
     "end_at": 1706875200
   },
@@ -1486,28 +751,7 @@
     "description": "最新のテクノロジーガジェットを紹介していきます。",
     "playlist_url": "https://media.xiii.isucon.dev/api/4/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 103,
-        "name": "椅子"
-      },
-      {
-        "id": 101,
-        "name": "周年記念"
-      },
-      {
-        "id": 29,
-        "name": "ピアノ"
-      },
-      {
-        "id": 67,
-        "name": "ニュース解説"
-      },
-      {
-        "id": 13,
-        "name": "マルチプレイ"
-      }
-    ],
+    "tags": null,
     "start_at": 1706832000,
     "end_at": 1706878800
   },
@@ -1528,28 +772,7 @@
     "description": "最新の美容機器を実際に試し、その効果や感想を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/2/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 78,
-        "name": "野球"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 100,
-        "name": "生誕祭"
-      },
-      {
-        "id": 55,
-        "name": "旅行記"
-      },
-      {
-        "id": 73,
-        "name": "科学"
-      }
-    ],
+    "tags": null,
     "start_at": 1706828400,
     "end_at": 1706871600
   },
@@ -1570,28 +793,7 @@
     "description": "美容師が春の新作ヘアスタイルやカラーを実際に公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 94,
-        "name": "UFO"
-      },
-      {
-        "id": 21,
-        "name": "視聴者参加"
-      },
-      {
-        "id": 13,
-        "name": "マルチプレイ"
-      },
-      {
-        "id": 54,
-        "name": "ダンス"
-      },
-      {
-        "id": 34,
-        "name": "夜ふかし"
-      }
-    ],
+    "tags": null,
     "start_at": 1706785200,
     "end_at": 1706853600
   },
@@ -1612,28 +814,7 @@
     "description": "アーティストがアコースティックセッションを公開。心温まる音楽の時間です。",
     "playlist_url": "https://media.xiii.isucon.dev/api/1/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 3,
-        "name": "生放送"
-      },
-      {
-        "id": 73,
-        "name": "科学"
-      },
-      {
-        "id": 59,
-        "name": "猫"
-      },
-      {
-        "id": 101,
-        "name": "周年記念"
-      },
-      {
-        "id": 88,
-        "name": "不動産"
-      }
-    ],
+    "tags": null,
     "start_at": 1708264800,
     "end_at": 1708275600
   },
@@ -1654,28 +835,7 @@
     "description": "日常で使える基本的な英会話フレーズを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/8/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 75,
-        "name": "コメディ"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      },
-      {
-        "id": 8,
-        "name": "レトロゲーム"
-      },
-      {
-        "id": 102,
-        "name": "サプライズ"
-      },
-      {
-        "id": 92,
-        "name": "手相"
-      }
-    ],
+    "tags": null,
     "start_at": 1708261200,
     "end_at": 1708275600
   },
@@ -1696,28 +856,7 @@
     "description": "子育ての専門家が、子供の健康や教育に関する悩みや質問に答えます。",
     "playlist_url": "https://media.xiii.isucon.dev/api/11/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 52,
-        "name": "ワークアウト"
-      },
-      {
-        "id": 74,
-        "name": "マジック"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 61,
-        "name": "釣り"
-      },
-      {
-        "id": 98,
-        "name": "コラボ配信"
-      }
-    ],
+    "tags": null,
     "start_at": 1708257600,
     "end_at": 1708264800
   },
@@ -1738,28 +877,7 @@
     "description": "実際に手作りのアクセサリーを作りながら、その方法やコツを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/6/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 96,
-        "name": "コンサート"
-      },
-      {
-        "id": 44,
-        "name": "手芸"
-      },
-      {
-        "id": 15,
-        "name": "ゲーム解説"
-      },
-      {
-        "id": 57,
-        "name": "キャンプ"
-      },
-      {
-        "id": 18,
-        "name": "新情報発表"
-      }
-    ],
+    "tags": null,
     "start_at": 1706785200,
     "end_at": 1706828400
   },
@@ -1780,28 +898,7 @@
     "description": "日常で使える基本的な英会話フレーズを紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 29,
-        "name": "ピアノ"
-      },
-      {
-        "id": 34,
-        "name": "夜ふかし"
-      },
-      {
-        "id": 89,
-        "name": "キャリア"
-      },
-      {
-        "id": 76,
-        "name": "スポーツ"
-      },
-      {
-        "id": 84,
-        "name": "起業"
-      }
-    ],
+    "tags": null,
     "start_at": 1706774400,
     "end_at": 1706814000
   },
@@ -1822,28 +919,7 @@
     "description": "我が家の子猫の成長記録を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/5/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 35,
-        "name": "日常話"
-      },
-      {
-        "id": 39,
-        "name": "手料理"
-      },
-      {
-        "id": 52,
-        "name": "ワークアウト"
-      },
-      {
-        "id": 60,
-        "name": "犬"
-      },
-      {
-        "id": 46,
-        "name": "映画レビュー"
-      }
-    ],
+    "tags": null,
     "start_at": 1708250400,
     "end_at": 1708261200
   },
@@ -1864,28 +940,7 @@
     "description": "アウトドア好きが集まるBBQの様子を公開し、楽しみ方やおすすめの食材を紹介します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/10/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/yoru.webp",
-    "tags": [
-      {
-        "id": 35,
-        "name": "日常話"
-      },
-      {
-        "id": 74,
-        "name": "マジック"
-      },
-      {
-        "id": 58,
-        "name": "ペットと一緒"
-      },
-      {
-        "id": 39,
-        "name": "手料理"
-      },
-      {
-        "id": 29,
-        "name": "ピアノ"
-      }
-    ],
+    "tags": null,
     "start_at": 1706767200,
     "end_at": 1706817600
   },
@@ -1906,28 +961,7 @@
     "description": "絵を描きながら、リラックスしたトークを楽しんでください。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 60,
-        "name": "犬"
-      },
-      {
-        "id": 67,
-        "name": "ニュース解説"
-      },
-      {
-        "id": 71,
-        "name": "心理学"
-      },
-      {
-        "id": 55,
-        "name": "旅行記"
-      },
-      {
-        "id": 5,
-        "name": "初心者歓迎"
-      }
-    ],
+    "tags": null,
     "start_at": 1706763600,
     "end_at": 1706832000
   },
@@ -1948,28 +982,7 @@
     "description": "実際の絵画の制作過程をリアルタイムで公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/1/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 53,
-        "name": "ヨガ"
-      },
-      {
-        "id": 11,
-        "name": "アクションゲーム"
-      },
-      {
-        "id": 22,
-        "name": "音楽ライブ"
-      },
-      {
-        "id": 26,
-        "name": "歌配信"
-      },
-      {
-        "id": 103,
-        "name": "椅子"
-      }
-    ],
+    "tags": null,
     "start_at": 1708250400,
     "end_at": 1708272000
   },
@@ -1990,28 +1003,7 @@
     "description": "我が家の愛犬の日常や生活の様子を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/8/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/BBB.webp",
-    "tags": [
-      {
-        "id": 8,
-        "name": "レトロゲーム"
-      },
-      {
-        "id": 76,
-        "name": "スポーツ"
-      },
-      {
-        "id": 38,
-        "name": "お料理配信"
-      },
-      {
-        "id": 4,
-        "name": "アドバイス"
-      },
-      {
-        "id": 99,
-        "name": "記念配信"
-      }
-    ],
+    "tags": null,
     "start_at": 1706749200,
     "end_at": 1706785200
   },
@@ -2032,28 +1024,7 @@
     "description": "朝の時間をヨガでリフレッシュ！初心者も大歓迎！",
     "playlist_url": "https://media.xiii.isucon.dev/api/5/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/isucon12_final.webp",
-    "tags": [
-      {
-        "id": 5,
-        "name": "初心者歓迎"
-      },
-      {
-        "id": 40,
-        "name": "レシピ紹介"
-      },
-      {
-        "id": 28,
-        "name": "ギター"
-      },
-      {
-        "id": 99,
-        "name": "記念配信"
-      },
-      {
-        "id": 49,
-        "name": "メイク"
-      }
-    ],
+    "tags": null,
     "start_at": 1708246800,
     "end_at": 1708250400
   },
@@ -2074,28 +1045,7 @@
     "description": "我が家の子猫の成長記録を公開します。",
     "playlist_url": "https://media.xiii.isucon.dev/api/9/playlist.m3u8",
     "thumbnail_url": "https://media.xiii.isucon.dev/timewarp.webp",
-    "tags": [
-      {
-        "id": 48,
-        "name": "ファッション"
-      },
-      {
-        "id": 2,
-        "name": "ゲーム実況"
-      },
-      {
-        "id": 18,
-        "name": "新情報発表"
-      },
-      {
-        "id": 89,
-        "name": "キャリア"
-      },
-      {
-        "id": 66,
-        "name": "DIY電子工作"
-      }
-    ],
+    "tags": null,
     "start_at": 1706727600,
     "end_at": 1706774400
   }
